app.js

```diff
@@ -1,83 +1,83 @@
-let collection = [];
+// ----- LIBRARY CLASS -----
 
-// -----CREATE A FUNCTION WHICH ADDS A NEW OBJECT TO THE ARRAY-----
-// ARRAY SHOULD GET THE INFORMATION FROM THE FORM INPUT VALUE
-const bookData = document.querySelector('form');
-const addBtn = document.querySelector('.add__add-btn');
+class Library {
+  constructor(collection) {
+    this.collection = collection;
+  }
 
-const bookTemplate = document.querySelector('.books__template');
-const bookContainer = document.querySelector('.container');
+  addObjectToArray() {
+    const bookData = document.querySelector('form');
+    const title1 = bookData.book_title.value;
+    const author1 = bookData.book_author.value;
+
+    const book = {
+      title: title1,
+      author: author1,
+    };
```

Comment:
[Optional]

- [ ] You can give unique id property for each book to make it easier to remove books.